crates/typst/src/layout/grid/mod.rs

```diff
@@ -185,8 +185,8 @@ pub struct GridElem {
 
     /// How to fill the cells.
     ///
-    /// This can be a color or a function that returns a color. The function is
-    /// passed the cells' column and row index, starting at zero. This can be
+    /// This can be a color or a function that returns a color. The function
+    /// gets the cells' column and row index, starting from zero. This can be
```

Comment:
I would stick with “is passed” here or maybe use “receives”.

Reply:
Receives! Of course! I can't believe I didn't think of that. I like it a lot more than either of the other two. @PgBiel?

Reply:
I think "receives" is good.

Reply:
Sure

Reply:
Done.